### ISSUE TITLE
[TW-1588] Problem with the included export scripts

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1614]
_Scott Carter on 2015-03-30T14:09:41Z says:_ 

 When i try to use the included export-html script, I get the following output:

<html>
  <body>
    <table>
      <thead>
        <tr>
          <td>ID</td>
          <td>Pri</td>
          <td>Description</td>
          <td>Project</td>
          <td>Due</td>
        </tr>
      </thead>
      <tbody>
garbage after JSON object, at character offset 246 (before "{"id":2,"description...") at /usr/share/perl5/JSON.pm line 171.

The other export scripts produce similar output.

It doesn't matter if I call the script by itself or pipe output from task to the script, I get the same message.  

I haven't tried it on previous versions, so I'm not sure if this is an issue that's been carried forward or not.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-03-30T14:09:41Z
Modified: 2015-03-31T05:33:59Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2015-03-30T19:21:42Z says:_ 

 This was fixed with [bc335e8075aab42de0c09fe8fc4133178a0d0810|https://git.tasktools.org/projects/TM/repos/task/commits/bc335e8075aab42de0c09fe8fc4133178a0d0810] for the export-html script. The others will need to be checked and fixed as well. Leaving this issue open.

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2015-03-31T05:33:59Z says:_ 

 Fixed with [3f394fa1642ff6990c91dcc8bd256fdd2c52a808|https://git.tasktools.org/projects/TM/repos/task/commits/3f394fa1642ff6990c91dcc8bd256fdd2c52a808]

Thank you for reporting this Scott!
